src/udiskslinuxmanager.c

```diff
@@ -1161,7 +1167,9 @@ static void
 manager_iface_init (UDisksManagerIface *iface)
 {
   iface->handle_loop_setup = handle_loop_setup;
+#ifdef HAVE_MDRAID
```

Comment:
Instead of leaving D-Bus methods unimplemented it's better to handle all method calls and only return error message within the handler with a proper error code stating that this feature has been intentionally disabled during compilation. It makes debugging on client side easier.

From D-Bus and C API perspective we should never hide any symbols conditionally.